### ISSUE TITLE
Hello world example of OpenTelemetry exporting data to New Relic

### ISSUE THREAD

**Issue by twaremike** [newrelic/newrelic-telemetry-sdk-dotnet#140]
Please provide a trivial c# console application that exports a relatively simple trace and a couple spans. After hours of struggle I'm unable to get this library to export any open telemetry traces/spans to the new relic platform.

**Comment by nrcventura** [comment]
@twaremike, thank you for this suggesstion. This sounds like a great idea.

**Comment by nr-ahemsath** [comment]
Hi @twaremike.  We should be able to add a simple console sample application pretty soon.  In the meantime, could you elaborate on the difficulties you are experiencing getting the sample web applications to send data to New Relic?  Maybe we need to improve our documentation of the sample apps.

**Comment by twaremike** [comment]
@nr-ahemsath Thanks for the sample app, the debug logging level makes clear what's wrong. It's a bad API key ( was using a rest api key starting with NRRA-*). I will make a feature to check config to prevent this sort of problem, and perhaps better docs around where & which key to use.

**Comment by twaremike** [comment]
@nr-ahemsath I noticed in the hello world example that you're simply waiting 10 seconds for the trace to be sent to New Relic. Is there any way to explicitly flush this data out to New Relic instead of just waiting? Perhaps a flush method would be good feature?

**Comment by twaremike** [comment]
I also noticed that the spans from the hello world get reported as errors, but there is no error with the HTTP request.
<img width="1220" alt="sample-span-error" src="https://user-images.githubusercontent.com/67486876/96651398-987de700-12e9-11eb-88aa-926f66b1da01.png">

**Comment by nr-ahemsath** [comment]
That's interesting with the span being reported as an error, I did not see that behavior when I tested it.  However, I wrote and tested it last Friday, and then Monday I updated it to OTel 0.7...I tested the app to make sure it ran, but I didn't go look at the spans in the UI.  I just ran it myself and sure enough, the spans are being marked as errors.  I'll try to figure out why.  Thanks for bringing this to our attention.
